backend/index.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let sessionCounter = 0;
+let counter;
+
+try {
+  fs.existsSync("temp/my-counter-visited-times.txt")
+    ? (counter = fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"))
+    : (counter = 0);
+} catch (e) {
+  console.log(e);
+}
+
+app.get("/", (req, res) => {
+  sessionCounter++;
+  counter = parseInt(counter);
+  res.send(
```

Comment:
Please check if you have responded or did all the comments left by the reviewer, you missed this one :P https://github.com/Zhenyatta/trello-tasks/pull/2#discussion_r1082408357

I meant to write something e.g. `res.status(200).send(....` if you think that it's a bad idea or something is blocking you from doing so you can write your thoughts under my comment and we can discuss it :3 
There is a trick I like to use not to miss the comments, when I am fixing the comments from my PRs AFTER FIXING the thing written in comment I add 👍 emoji to it or I write something extra and the next time I scroll to that comment I ignore it if it has the emoji.